### PR TITLE
New column for generic tracker

### PR DESCRIPTION
**Pull request by Fluffyhairedguy**
Adding an order received column for the generic tracker so you can sort by it and see the most recent items received. Progressive items will have the number of the last one acquired. Handles new games where item list doesn't exist yet, and tested with multiple games.

### CODE REVIEW

**Review by Berserker66 — CHANGES_REQUESTED** [review]

File: WebHostLib/tracker.py [hunk 1]
```diff
@@ -521,11 +521,17 @@ def getPlayerTracker(tracker: UUID, tracked_team: int, tracked_player: int):
 
     else:
         checked_locations = multisave.get("location_checks", {}).get((tracked_team, tracked_player), set())
+        player_received_items = {}
+        if len(multisave) > 0:
```
[the remark this answers is not in the document]
Comment (Fluffyhairedguy) [reply WebHostLib/tracker.py]: Fixed, didn't realize I was trying to do the same thing as two lines above.

File: WebHostLib/tracker.py [hunk 2]
```diff
@@ -521,11 +521,17 @@ def getPlayerTracker(tracker: UUID, tracked_team: int, tracked_player: int):
...
+            for key in multisave['received_items']:
+                if key[1] == tracked_player and len(multisave['received_items'][key]) > 0:
```
[the remark this answers is not in the document]
Comment (Fluffyhairedguy) [reply WebHostLib/tracker.py]: Fixed and simplified

File: WebHostLib/tracker.py [hunk 3]
```diff
@@ -521,11 +521,17 @@ def getPlayerTracker(tracker: UUID, tracked_team: int, tracked_player: int):
...
+                    for NetworkItem in multisave['received_items'][key]:
```
[the remark this answers is not in the document]
Comment (Fluffyhairedguy) [reply WebHostLib/tracker.py]: Fixed, was not intentional

File: WebHostLib/tracker.py [hunk 4]
```diff
@@ -521,11 +521,17 @@ def getPlayerTracker(tracker: UUID, tracked_team: int, tracked_player: int):
...
+                        player_received_items[NetworkItem.item] = multisave['received_items'][key].index(NetworkItem) + 1
```
[the remark this answers is not in the document]
Comment (Fluffyhairedguy) [reply WebHostLib/tracker.py]: Now I learned about enumerate, it's exactly what I needed, fixed!

**Pull request merged**